### PR TITLE
[Console] Document how to exclude messages from output using SymfonyStyle

### PR DESCRIPTION
**Pull request by chalasr**
This adds documentation for https://github.com/symfony/symfony/pull/20586, any suggestion will be really welcomed.

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
@chalasr I liked this contribution a lot ... but I wanted to reword it slightly to better explain the `stdout` and `stderr` differences because `stderr` is still very confusing and a lot of developers think it's only useful to output error messages.

**Comment by chalasr** [comment]
@javiereguiluz You did right, it was a bit unclear. Thanks for that!

**Comment by javiereguiluz** [comment]
Any other review for this before merging it? Thanks!

File: console/style.rst [hunk 1]
```diff
@@ -373,3 +373,35 @@ of your commands to change their appearance::
...
+
+    // Write to the error output
+    $io->getErrorStyle()->warning('Debugging information or errors');
+
+.. note::
+
+    If you create a ``SymfonyStyle`` instance with an ``OutputInterface`` object
+    that is not an instance of :class:`Symfony\\Component\\Console\\Output\\ConsoleOutputInterface`,
+    the ``getErrorStyle()`` method will have no effect and the returned object
+    will still output to the standard output instead of the error output.
```
Comment (HeahDude) [inline console/style.rst]: "will still write" to avoid the repetition?

**Review by HeahDude — APPROVED** [review]
Left a minor comment but 👍

**Comment by javiereguiluz** [comment]
It's merged! Thanks @chalasr.

**Comment by chalasr** [comment]
and thank you @javiereguiluz :)

**Pull request merged**